### PR TITLE
v. 1.4.0 release notes

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+* :jirabug:`K8SPXC-125`: Percona XtraDB Cluster 8.0 is now supported
+
+Improvements
+================================================================================
+
+* :jirabug:`K8SPXC-221`: Operator now updates observedGeneration status message to allow better monitoring of the cluster rollout or backup/restore process
+* :jirabug:`K8SPXC-153`: S3 protocol credentials are now masked in logs during the PXC backup & restore process
+* :jirabug:`K8SPXC-234`: A special script to execute Graceful Shutdown of the cluster is now available. It is `triggered <../pause.html>`_ by the ``spec.pause`` key in the ``deploy/cr.yaml`` file.
+* :jirabug:`K8SPXC-232` and :jirabug:`K8SPXC-233`: Two Crash Recovery methods were developed: the automated but lossy ref:`recovery-bootstrap` and the manual but lossless :ref:`recovery-object-surgery`.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst]: it is about documentation only.

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+:Installation: `Installing Percona Kubernetes Operator for PXC <https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html#installation>`_
+
+New Features
+================================================================================
+
+* :jirabug:`K8SPXC-125`: Percona XtraDB Cluster 8.0 is now supported
+
+Improvements
+================================================================================
+
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst]: please add K8SPXC-100

Comment (fiowro) [reply source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst:22]: Done, but I'm still not sure about the description. 
1) Is it PXC 8.0 only? 
2) Should this feature also be mentioned in the graceful shutdown or crash recovery doc?

Comment (delgod) [reply source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst:22]: it is for both 5.7 and 9.0, the description is ok

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+* :jirabug:`K8SPXC-95`: Amazon Elastic Container Service for Kubernetes (EKS)
+  was added to the list of the officially supported platforms
+* The OpenShift Container Platform 4.3 is now supported.
+* :jirabug:`K8SPXC-172`: Full data-at-rest encryption available in PXC 8.0 is now supported by the Operator. This feature is implemented with the help of the ``keyring_vault`` plugin which ships with PXC 8.0.  By utilizing `Vault <https://www.vaultproject.io>`_ we enable our customers to follow best practices with encryption in their environment.
+
+Improvements
+================================================================================
+
+* :jirabug:`K8SPXC-221`: Operator now updates observedGeneration status message to allow better monitoring of the cluster rollout or backup/restore process
+* :jirabug:`K8SPXC-232` and :jirabug:`K8SPXC-233`: Two Crash Recovery methods were developed: the automated but lossy ref:`recovery-bootstrap` and the manual but lossless :ref:`recovery-object-surgery`.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst:24]:
```suggestion
* :jirabug:`K8SPXC-232` and :jirabug:`K8SPXC-233`: Two Crash Recovery methods were documented: the automated but lossy ref:`recovery-bootstrap` and the manual but lossless :ref:`recovery-object-surgery`.
```

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+* :jirabug:`K8SPXC-95`: Amazon Elastic Container Service for Kubernetes (EKS)
+  was added to the list of the officially supported platforms
+* The OpenShift Container Platform 4.3 is now supported.
+* :jirabug:`K8SPXC-172`: Full data-at-rest encryption available in PXC 8.0 is now supported by the Operator. This feature is implemented with the help of the ``keyring_vault`` plugin which ships with PXC 8.0.  By utilizing `Vault <https://www.vaultproject.io>`_ we enable our customers to follow best practices with encryption in their environment.
+
+Improvements
+================================================================================
+
+* :jirabug:`K8SPXC-221`: Operator now updates observedGeneration status message to allow better monitoring of the cluster rollout or backup/restore process
+* :jirabug:`K8SPXC-231`: A :ref:`graceful stop/start of the cluster<operator-pause>` is now implemented, and for situations when it was not done, :ref:`crash recovery methods<operator-recovery>` are available as well.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst:24]: it is just documentation issue

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+* :jirabug:`K8SPXC-213`: A special PXC debug image is now available. It avoids restarting on fail and contains additional tools useful for debugging.
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst:25]: link to documentation should be added

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst (hunk 4 above)
Comment (delgod) [reply source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst:24]: I think we don't include doc tickets into RN

File: source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+================================================================================
+
+* :jirabug:`K8SPXC-221`: Operator now updates observedGeneration status message to allow better monitoring of the cluster rollout or backup/restore process
+* :jirabug:`K8SPXC-213`: A special :ref:`PXC debug image<debug-images>` is now available. It avoids restarting on fail and contains additional tools useful for debugging.
+* :jirabug:`K8SPXC-100`: The Operator now implements the crash tolerance on the one member crash. The implementation is based on starting Pods with ``mysqld --wsrep_recover`` command if there was no graceful shutdown.
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`K8SPXC-153`: S3 protocol credentials were not masked in logs during the PXC backup & restore process
```
Comment (delgod) [inline source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst:30]: please add https://jira.percona.com/browse/K8SPXC-261 also

Comment (delgod) [reply source/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.4.0.rst:31]: K8SPXC-263 still needed

**Pull request merged**